library/bz2.po

```diff
@@ -276,28 +363,39 @@ msgid ""
 "this case, the next call to :meth:`~.decompress` may provide *data* as "
 "``b''`` to obtain more of the output."
 msgstr ""
+"Si *max_length* est positif, renvoie au moins *max_length* octets de données "
+"compressées. Si la limite est atteinte et que d'autres sorties peuvent être "
+"produites, l'attribut :attr:`~.needs_input` sera configuré sur ``False``. "
```

Comment:
```suggestion
"produites, l'attribut :attr:`~.needs_input` est positionné sur ``False``. "
```

Reply:
Là, encore tu as un problème de temps : "will be verb" n'est pas du présent !
Pas de problème avec "positionné" ;)

Reply:
Les anglais utilisent beaucoup le futur pour indiquer une vérité quand en francais on préfère le présent. J'approuve la proposition de @christopheNan.

D'ailleurs on applique ça sur toute la traduction, il serait bon de le rajouter au README ?